[2024-09-13T03:55:44.356+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-13T03:55:44.529+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v3.insert_into_table scheduled__2024-09-08T00:00:00+00:00 [queued]>
[2024-09-13T03:55:44.582+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v3.insert_into_table scheduled__2024-09-08T00:00:00+00:00 [queued]>
[2024-09-13T03:55:44.673+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-13T03:55:45.134+0000] {taskinstance.py:2879} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-09-08 00:00:00+00:00
[2024-09-13T03:55:45.629+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=466) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-13T03:55:45.798+0000] {standard_task_runner.py:72} INFO - Started process 507 to run task
[2024-09-13T03:55:45.918+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v3', 'insert_into_table', 'scheduled__2024-09-08T00:00:00+00:00', '--job-id', '558', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator_dag.py', '--cfg-path', '/tmp/tmpk8iwvpp4']
[2024-09-13T03:55:45.922+0000] {standard_task_runner.py:105} INFO - Job 558: Subtask insert_into_table
[2024-09-13T03:55:47.285+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_operator_v3.insert_into_table scheduled__2024-09-08T00:00:00+00:00 [running]> on host 01a3c7586b71
[2024-09-13T03:55:47.880+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v3' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-08T00:00:00+00:00'
[2024-09-13T03:55:47.884+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-13T03:55:47.933+0000] {sql.py:282} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2024-09-08', 'dag_with_postgres_operator_v3')
        
[2024-09-13T03:55:47.960+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-13T03:55:47.999+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-09-13T03:55:48.057+0000] {sql.py:509} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2024-09-08', 'dag_with_postgres_operator_v3')
        , parameters: None
[2024-09-13T03:55:48.066+0000] {sql.py:518} INFO - Rows affected: 1
[2024-09-13T03:55:48.091+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-13T03:55:48.093+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v3, task_id=insert_into_table, run_id=scheduled__2024-09-08T00:00:00+00:00, execution_date=20240908T000000, start_date=20240913T035544, end_date=20240913T035548
[2024-09-13T03:55:48.165+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-13T03:55:48.232+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-13T03:55:48.243+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
